### PR TITLE
Adopt apprepository-controller to the changes in kubeapps/chartsvc

### PR DESCRIPTION
**Pull request by murali-reddy**
New binary chart-repo supports both sync and delete commands. So use sync command in cronjob and jobs created for repo sync

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -424,7 +424,7 @@ func jobSpec(apprepo *apprepov1alpha1.AppRepository) batchv1.JobSpec {
 					{
 						Name:    "sync",
 						Image:   repoSyncImage,
```
Comment (prydonius) [inline controller.go:434]: I think we probably want to also change the docker repo name to `kubeapps/chart-repo` or something. Now that @ngtuna's PR is merged I can push it there.

Comment (murali-reddy) [reply controller.go:434]: Sure. Please push the image and let me know, I can actually test end-to-end with proper image.

Comment (prydonius) [reply controller.go:434]: Sorry, got distracted. Just pushed kubeapps/chart-repo:latest :)

**Comment by murali-reddy** [comment]
@prydonius @ngtuna added support for both sync and deletion.

**Review by prydonius — APPROVED** [review]
lgtm, thanks!

**Pull request merged**